mne/viz/tests/test_raw.py

```diff
@@ -421,6 +423,12 @@ def test_plot_raw_psd():
     raw.plot_psd(picks=picks, average=False)
     raw.plot_psd(picks=picks, average=True)
     plt.close('all')
+    old_log_level = set_log_level(logging.CRITICAL)
```

Comment:
This was required for the set_channel_types not to fail out the test. But it feels wrong.

Reply:
Because of an emitted warning? Pass verbose='error' to the function call that otherwise emits a warning to stop it

Reply:
`set_channel_types` doesn't have a verbose option
https://mne.tools/dev/generated/mne.io.Raw.html#mne.io.Raw.set_channel_types

Reply:
Well that was less of a scary change than I thought. Thanks for the push.